### ISSUE TITLE
Mediapipe Iris hanging after the first frame

### ISSUE THREAD

**Issue by ctippur** [google/mediapipe#1591]
I seem to be stuck with the same issue - https://github.com/google/mediapipe/issues/850. I have seen the comments and looks like there are some suggestions but some reservations as well. 
I am using a video file as input. 
1. Why am I observing this behavior? I dont see any issues with the video. I have another similar video where this is not observed. 
2. What is the most graceful way to handle this? How do I skip a frame. 
In layman terms, all I want to do is, if I encounter an issue with a particular frame in the video stream, skip it. 

Appreciate any pointers.

**Comment by ianni67** [comment]
In my knowledge, this issue has been solved in the last version of mediapipe. In particular, look at https://github.com/google/mediapipe/blob/master/mediapipe/examples/desktop/demo_run_graph_main_gpu.cc#L98-L106.

**Comment by rmothukuru** [comment]
@ctippur,
Can you please let us know if you have tried latest version of MediaPipe as stated in the [above comment](https://github.com/google/mediapipe/issues/1591#issuecomment-775761459) and if the issue still persists? Thanks!

**Comment by rmothukuru** [comment]
@ctippur,
Can you please respond to the above comment? Thanks!

**Comment by ctippur** [comment]
Apologies. I will get back on this today.

**Comment by ctippur** [comment]
I just tested. I seem to be getting this error

```
Failed to run the graph: CalculatorGraph::Run() failed in Run: 
Calculator::Process() for node "facelandmarkfrontcpu__facedetectionfrontcpu__NonMaxSuppressionCalculator" failed: NonMaxSuppression received empty detections, but detections are required for this pipeline.
```

**Comment by ctippur** [comment]
I may have hit this bug issue https://github.com/google/mediapipe/issues/1616 ?

**Comment by ctippur** [comment]
Any pointers to resolve this?

**Comment by ctippur** [comment]
> @ctippur,
> Can you please respond to the above comment? Also, can you please let us know in which platform you are executing **`Iris Solution`**?Thanks!

I am using the desktop platform.

**Comment by ctippur** [comment]
Apologies. I had to add the missing DetectionLetterboxRemovalCalculator calculator to the build. I am able to process the video now. We can close this issue. Thanks for the help.

**Comment by sgowroji** [comment]
Glad to see you resolved.
